### ISSUE TITLE
Existem informações faltando no webhook

### ISSUE THREAD

**Issue by andreladocruz** [deveduzz/webhook-eduzz#1]
Olá @deveduzz,

Sou o responsável técnico pela plataforma Digital Manager Guru (https://digitalmanager.guru).

Apesar do webhook ter melhorado bastante, ainda vejo a necessidade de algumas informações que estão ausentes:

- tipo de post (produtor e afiliado) - tipo de informação (de produtor ou afiliado) sendo enviada;
- valor da eduzz (produtor) - valor cobrado pela eduzz naquela venda;
- valor líquido (produtor) - (trans_value - aff_value - eduzz_value);
- Nome do Produtor (produtor e afiliado) - importante para emissão de nota fiscal;
- Documento do produtor (produtor e afiliado) - importante para emissão de nota fiscal;
- Email do produtor (produtor e afiliado) - importante para emissão de nota fiscal;

Existe a possibilidade de vocês adicionarem este tipo de informação?

Abraços,

**Comment by moskoweb** [comment]
Opa rapaz, eu fiz um Pull Request com as informações mais atualizadas.

**Comment by leonardoborlot** [comment]
Uma outra coisa que falta é ter mais de um canal pro webhook por produto.


Em 8 de fev de 2018 04:32, "Alan Mosko" <notifications@github.com> escreveu:

**Comment by moskoweb** [comment]
Concordo, deveria de ter realmente, pois tem dois lugares para receber um postback, cada um com uma coisa diferente.

Tem momento que tem de fazer uma pseudo gambiarra aceitando pela entrega customizada, que permite mais de um, mais vai menos informações.

**Comment by andreladocruz** [comment]
A integração do Digital Manager Guru foi através do Connect (originalmente usado para autoresponders) justamente por conta dessa limitação.

**Comment by andreladocruz** [comment]
@moskoweb 
Já pegamos todas as informações das recorrências através de nossa integração. =) 
Eles possuem dois tipos de recorrência (antiga e nova) ativos.
Então, cuidado pois você pode estar pegando apenas um dos tipos. =P 
A integração deles não é uma das melhores do mercado, sendo BEEM sincero.

**Comment by moskoweb** [comment]
Pois se, está bem falha a documentação também, estou usando postback da forma que conversei com um cara do TI de lá, mas penso em usar o Connect também, para uma segurança maior.

**Comment by andreladocruz** [comment]
@moskoweb 
O Connect dependente de integração do lado deles... =) 
A não ser que você usa a integração do Guru e coloque uma URL sua para receber os posts. 
hahahhahahahaah
#BurlandoOSistema

**Comment by moskoweb** [comment]
kkkkkkk oloko, ai sim.
